api/src/lambda_handler.py

```diff
@@ -0,0 +1,9 @@
+import serverless_wsgi
+from hyp3_api import connexion_app
+
+
+serverless_wsgi.TEXT_MIME_TYPES.append('application/problem+json')
+
+
+def handler(event, context):
+    return serverless_wsgi.handle_request(connexion_app.app, event, context)
```

Comment:
should we write a test for this?  at least invoke with a typical api gateway event and check that we get a 404 or 401 resopnse back?

Reply:
I don't know that this really needs a test, it's just a call to a library function.